nci/lib/watcher.rb

```diff
@@ -260,10 +260,23 @@ def run
         #end
       #end
 
-      bump_version
-
-      cmd.run('git --no-pager diff')
-      cmd.run("git commit -a -vv -m 'New release'")
+      # check if future_is_early and make sure correct series is applied and
+      # bump both release_* branches if required
+
+      if NCI.future_is_early
```

Comment:
`if NCI.future_series` I think.

Otherwise we bump only the **current** when the **future** is set but not marked early.